packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -174,20 +174,26 @@ class PersonalDetailsForm extends React.Component {
         }
 
         if (values.phone) {
+            // minimum characters required is 9 including (+) sign
+            // phone_trim uses regex that trims (+) sign
+            // minimum characters required w/o (+) sign is 8 characters.
             const min_phone_number = 8;
             const max_phone_number = 35;
             const phone_trim = values.phone.replace(/\D/g, '');
 
-            if (!validPhone(values.phone)) {
-                errors.phone = localize(
-                    'Please enter a valid phone number, including the country code (e.g. +15417541234)'
-                );
+            if (!validLength(phone_trim, { min: min_phone_number, max: max_phone_number })) {
+                errors.phone = localize('You should enter {{min}}-{{max}} numbers.', {
+                    min: min_phone_number,
+                    max: max_phone_number,
+                });
             } else if (!validCountryCode(this.props.residence_list, values.phone)) {
                 errors.phone = localize(
                     'Please enter a valid phone number, including the country code (e.g +15417541234).'
                 );
-            } else if (!validLength(phone_trim, { min: min_phone_number, max: max_phone_number })) {
-                errors.phone = localize('You should enter 8-35 characters.');
+            } else if (!validPhone(values.phone)) {
+                errors.phone = localize(
+                    'Please enter a valid phone number, including the country code (e.g. +15417541234)'
+                );
```

Comment:
can we store `localize('Please enter a valid phone number, including the country code (e.g. +15417541234)');` in a variable since it's used twice here?

Reply:
We can add it to `form-error-messages.js` instead and call it using `getErrorMessages` from `@deriv/shared`.

Reply:
@zaki-binary there's another phone error message there that is being used, let's handle that refactoring separately